package/src/components/DataTable/DataTable.js

```diff
@@ -0,0 +1,350 @@
+import React from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TextField,
+  TableRow,
+  Toolbar,
+  makeStyles,
+  Typography
+} from "@material-ui/core";
+import ChevronLeftIcon from "mdi-material-ui/ChevronLeft";
+import ChevronRightIcon from "mdi-material-ui/ChevronRight";
+import Button from "../Button";
+import Select from "../Select";
+import ActionMenu from "../ActionMenu";
+
+const useStyles = makeStyles((theme) => ({
+  pagination: {
+    color: theme.palette.colors.coolGrey500,
+    letterSpacing: 0.28,
+    paddingTop: theme.spacing(2)
+  },
+  tableBody: {
+
+  },
+  tableRowOdd: {
+    backgroundColor: theme.palette.colors.black02
+  },
+  tableHead: {
+    fontWeight: theme.typography.fontWeightSemiBold,
+    letterSpacing: 0.5,
```

Comment:
`h5LetterSpacing` variable

Reply:
@cassytaylor should the table head just be an H5 (in the display, but as a literal H5 element) and If so, should all H5's be semi-bold?

(trying to reduce one-off font variants like these)